jsonrpc/dispatcher.go

```diff
@@ -448,9 +449,14 @@ func (d *Dispatcher) getNextNonce(address types.Address, number BlockNumber) (ui
 		return 0, err
 	}
 	acc, err := d.store.GetAccount(header.StateRoot, address)
-	if err != nil {
+	if err != nil && errors.Is(err, ErrStateNotFound) {
```

Comment:
You don't need to do nil-check to avoid passing nil to errors.Nil

```suggestion
	if errors.Is(err, ErrStateNotFound) {
```

Reply:
Fixed 👍 

33e84c7